docs/whatisk8s.md

```diff
@@ -118,7 +118,7 @@ Kubernetes satisfies a number of common needs of applications running in product
 * [support for introspection and debugging](user-guide/introspection-and-debugging.md), and
 * [identity and authorization](admin/authorization.md).
 
-This provides the simplicity of Platform as a Service (PaaS) with the flexibility of Infrastructure as a Service (IaaS).
+This provides the simplicity of Platform as a Service (PaaS) with the flexibility of Infrastructure as a Service (IaaS), an enables portability across infrastructure providers.
```

Comment:
This is an overview, which is already getting long.

I agree we need a separate doc.